src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -182,6 +182,12 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
                     telemetry.TelemetryEvent[IsCentralPackageTransitivePinningEnabled] = isCentralPackageTransitivePinningEnabled;
                 }
 
+                bool auditEnabled = AuditUtility.ParseEnableValue(
```

Comment:
```suggestion
                bool auditEnabled = AuditUtility.ParseEnableValue(
                    _request.Project.RestoreMetadata?.RestoreAuditProperties,
                    _request.Project.FilePath,
                    _logger);
                telemetry.TelemetryEvent[AuditEnabled] = auditEnabled;
```

Reply:
"enabled", "disabled" is what has been reported in the original telemetry for over a year now, so I don't want to change it since it'd require changing a lot of queries.